src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableList_1.Node.cs

```diff
@@ -493,11 +481,7 @@ internal Node ReplaceAt(int index, T value)
             /// Reverses the order of the elements in the entire <see cref="ImmutableList{T}"/>.
             /// </summary>
             /// <returns>The reversed list.</returns>
-            internal Node Reverse()
-            {
-                Contract.Ensures(Contract.Result<Node>() != null);
-                return this.Reverse(0, this.Count);
-            }
+            internal Node Reverse() => this.Reverse(0, this.Count);
```

Comment:
@JonHanna Although corefx generally omits `this.` it seems that System.Collections.Immutable was written with a different style guide. I wanted to preserve that style in this PR.